code/game/objects/structures/xeno.dm

```diff
@@ -130,6 +130,8 @@
 	if(builder)
 		linked_carrier = builder
 	RegisterSignal(src, COMSIG_MOVABLE_SHUTTLE_CRUSH, .proc/shuttle_crush)
+	RegisterSignal(get_turf(src), COMSIG_ATOM_ENTERED, .proc/trigger_hugger_trap) //Set up the trap signal on our turf
+
```

Comment:
Why

Reply:
Simplifies the code a bit; if you want it changed back I don't care too much, but this works perfectly fine

Reply:
Change it to actually register to the trap being crossed